src/components/marsPhotoViewer/ImageSelector.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import { getImagesFromApi, imageApiModel } from "../../api/imagesApi";
+import "./ImageSelector.scss";
+
+export function ImageSelector(props: {
+    setState: React.Dispatch<React.SetStateAction<string>>,
+    date: string,
+    rover: string,
+    camera: string,
+}) {
+    const [imgUrl, setImgUrl] = useState<imageApiModel[]>();
+
+    useEffect(() => {
+        getImagesFromApi(props.date, props.rover, props.camera)
+            .then(data => setImgUrl(data))
+            .then(() => { if (imgUrl) props.setState(imgUrl[0].img_src) });
+    }, []);
```

Comment:
I think we will want to pass in date, rover and camera here to make the useEffect run when these change but can do this when we link up with the filters